package/MDAnalysis/analysis/base.py

```diff
@@ -316,6 +489,7 @@ def __init__(self, trajectory, verbose=False, **kwargs):
         self._trajectory = trajectory
         self._verbose = verbose
         self.results = Results()
+        super(AnalysisBase, self).__init__(self)
```

Comment:
this ugly super call signature was Python 2.x, I think we can use `super().__init__(self)` now

Reply:
Agree